<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaningDialog</class>
 <widget class="QDialog" name="CleaningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>清洗数据</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>文件</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rawDataPath_lineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>501</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="rawDataPath_pushButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>100</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>编码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>230</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>语种</string>
       </property>
      </widget>
      <widget class="QComboBox" name="rawDataEncoding_comboBox">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>230</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>UTF-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ANSI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UNICODE</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="rawDataLang_comboBox">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>230</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>中文</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>英文</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="rawDataType_comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>251</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>题录—知网—es5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>题录—GBT 7714-2015</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>题录—知网—NoteExpress</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startCleaning_pushButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>410</y>
         <width>150</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>开始清洗</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openbrowser_pushButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>打开网站</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
